### PR TITLE
[spark] Support global options via SQL conf

### PR DESCRIPTION
**Pull request by Zouxxyy**
### Purpose

Users can configure paimon option globally via `spark.paimon.x`, e.g.

`set spark.paimon.file.block-size=512M` to control the file block size globally

### CODE REVIEW

**Review by JingsongLi — COMMENTED** [review]
Why not just using `paimon.xxx`?

**Comment by Zouxxyy** [comment]
> Why not just using `paimon.xxx`?

Spark will only recognize sql conf starting with `spark.`

**Comment by Zouxxyy** [comment]
For example, if we want to configure hadoop conf to spark, we need to set like `spark.hadoop.x`

**Review by JingsongLi — COMMENTED** [review]
+1 after test pass.

File: paimon-spark/paimon-spark-common/src/main/scala/org/apache/paimon/spark/PaimonOptionHelper.scala [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+import org.apache.paimon.table.Table
+
+import org.apache.spark.sql.catalyst.SQLConfHelper
+
+import java.util.{HashMap => JHashMap, Map => JMap}
+
+import scala.collection.JavaConverters._
+
+object PaimonOptionHelper extends SQLConfHelper {
```
Comment (YannByron) [inline paimon-spark/paimon-spark-common/src/main/scala/org/apache/paimon/spark/PaimonOptionHelper.scala:29]: `Helper` typically represents a trait, not a object.

**Review by JingsongLi — APPROVED** [review]
Looks good to me!

**Review by YannByron — APPROVED** [review]

**Pull request merged**